cnxarchive/sql/migrations/20160622095217_add_modules_stateid_default.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+
+def up(cursor):
+    cursor.execute("""\
+ALTER TABLE modules
+  ALTER COLUMN stateid SET DEFAULT 1""")
+
```

Comment:
Yup, I checked, it takes an AccessExclusiveLock, which blocks parallel reads. This might be a bit tricky to rollout on production. Not really any other way to do it, though.